starter/script.js

```diff
@@ -1 +1,33 @@
-// put code here!
+ alert("Welcome My name is Sarah")
+  var name prompt("What is your name?")
```

Comment:
Watch your indentation! Things should only be indented to the right if they're "inside" something else. For example:

```
function yo(){
  alert("I'm indented because I'm inside the function")
}
if(x === 3){
  alert("I'm indented because I'm inside the if statement")
}
```